### DISCUSSION TITLE
Best way to create a clickable tooltip

### DISCUSSION THREAD

**Discussion by cubecleveland** [ariakit/ariakit#938] [Questions] [answered]
Hello, trying to find the best way to create a 'clickable tooltip'.
Is there a native way to do it using the Tooltip or popover...

Is there a code ample i can use ?
basically trying to achieve a situation where tooltip appears on hover and then stays open if muse is on it while accepting click events

**Comment by diegohaz** [comment, marked as answer]
It looks like you want a Hovercard. It's not documented yet, but it's going to be available on v2, which is currently in alpha.

You can see a minimal example here (things may change until the final release): https://codesandbox.io/s/2x8h8?module=src/hovercard.tsx
